src/idiom327ex.fz

```diff
@@ -0,0 +1,3 @@
+convert_to_lower_case is
+    s := "hELlO"
+    say "{s.lower_case}"
```

Comment:
Here you should include the exact code from the file `src/idiom327.fuzion` and just do initialization of fields before and output afterwards.